tests/test-cases/Group1-Docker-Commands/1-08-Docker-Logs.robot

```diff
@@ -173,9 +173,6 @@ Docker text logs
     Should Be Equal As Integers  ${rc}  0
 
 Docker logs with timestamps and since certain time
-    ${status}=  Get State Of Github Issue  2539
-    Run Keyword If  '${status}' == 'closed'  Fail  Test 1-8-Docker-Logs.robot needs to be updated now that Issue #2539 has been resolved
-    Log  Issue \#2539 is blocking implementation  WARN
     ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} pull busybox
     Should Be Equal As Integers  ${rc}  0
     ${rc}  ${containerID}=  Run And Return Rc And Output  docker %{VCH-PARAMS} create busybox /bin/sh -c 'a=0; while [ $a -lt 5 ]; do echo "line $a"; a=`expr $a + 1`; sleep 1; done;'
```

Comment:
the bottom of this test needs to be updated to remove the does not support '--since', but apparently it won't let me comment on code beyond the section it wants me to :)